api/src/utils/get-challenges.ts

```diff
@@ -0,0 +1,37 @@
+import { flatten } from 'lodash';
+
+// TODO: keeping curriculum in memory is handy if we want to field requests that
+// need to 'query' the curriculum, but if we force the client to handle
+// redirectToCurrentChallenge and, instead, only report the current challenge id
+// via the user object, then we should *not* store this so it can be garbage
+// collected.
+import curriculum from '../../../config/curriculum.json';
+import { SuperBlocks } from '../../../config/superblocks';
+import { type ChallengeNode } from '../../../client/src/redux/prop-types';
```

Comment:
```suggestion
```
Let's not import from the client. For one thing the types in there are often bad guesses and for another we want the api to be the source of truth for types.